jax/_src/numpy/lax_numpy.py

```diff
@@ -2543,17 +2543,41 @@ def _pad(array, pad_width, mode, constant_values, stat_length, end_values, refle
     return _pad_empty(array, pad_width)
 
   else:
```

Comment:
I do not think that we need these changes, they were merged already in #5229.

Reply:
Yes, I misunderstood.
I thought #5229 will be reverted and this new PR is merged after fixing the breaking changes.

I updated PR only modifying change log. PTAL:)